WORKSPACE

```diff
@@ -9,11 +10,10 @@ http_archive(
     sha256 = "d7dc12c1d5bc1a87474de8e3d17b7731a4dcebcfb8aa3990fe8ac7734ef12f2f",
 )
 
-http_archive(
+git_repository(
```

Comment:
https://docs.bazel.build/versions/main/external.html#repository-rules suggests a preference for `http_archive`